tests/lib/eslint/eslint.js

```diff
@@ -5442,257 +5442,6 @@ describe("ESLint", () => {
-                    files: {
-                        "rules.ts": `export default ${
-                            JSON.stringify({
-                                "no-undef": 2
-                            }, null, 2)};`,
-                        "package.json": typeCommonJS,
-                        "eslint.config.ts": configFileContent,
-                        "foo.js": "foo;"
-                    }
-                });
-
-                await teardown.prepare();
-
-                eslint = new ESLint({
-                    cwd,
-                    flags
-                });
-
-                const results = await eslint.lintFiles(["foo.js"]);
-
-                assert.strictEqual(await eslint.findConfigFile(), path.join(cwd, "eslint.config.ts"));
-                assert.strictEqual(results.length, 1);
-                assert.strictEqual(results[0].messages.length, 1);
-                assert.strictEqual(results[0].messages[0].severity, 2);
-                assert.strictEqual(results[0].messages[0].ruleId, "no-undef");
-
-            });
-
-            it("should load eslint.config.ts with CJS syntax, \"type\": \"module\" in nearest `package.json` and top-level await syntax (default and named imports)", async () => {
-
-                const cwd = getFixturePath("ts-config-files", "ts", "with-type-module", "top-level-await", "import-default-and-named");
-
-                const configFileContent = "import type { FlatConfig } from \"../../../../helper\";\nconst { default: rules, Level } = await import(\"./rules\");\n\nmodule.exports = [{ rules: { ...rules, semi: Level.Error } }] satisfies FlatConfig[];";
-
-                const teardown = createCustomTeardown({
-                    cwd,
-                    files: {
-                        "rules.ts": `import type { RulesRecord } from "../../../../helper";\nexport const enum Level {\nError = 2,\nWarn = 1,\nOff = 0,\n};\nexport default ${
-                            JSON.stringify({
-                                "no-undef": 2
-                            }, null, 2)} satisfies RulesRecord;`,
-                        "package.json": typeModule,
-                        "eslint.config.ts": configFileContent,
-                        "foo.js": "foo"
-                    }
-                });
-
-                await teardown.prepare();
-
-                eslint = new ESLint({
-                    cwd,
-                    flags
-                });
-
-                const results = await eslint.lintFiles(["foo.js"]);
-
-                assert.strictEqual(await eslint.findConfigFile(), path.join(cwd, "eslint.config.ts"));
-                assert.strictEqual(results.length, 1);
-                assert.strictEqual(results[0].messages.length, 2);
-                assert.strictEqual(results[0].messages[0].severity, 2);
-                assert.strictEqual(results[0].messages[1].severity, 2);
-                assert.strictEqual(results[0].messages[0].ruleId, "no-undef");
-
-            });
-
```

Comment:
I was thinking that we could keep these unit tests commented out with `/*` ... `*/` so we can quickly re-enable them when we upgrade to Jiti v2. If we delete the tests, we should probably also delete the unused fixtures.

Reply:
Good idea, I will comment them out instead.

Reply:
Do you want me to do the same with code inside `loadFlatConfigFile`?

Reply:
The changes in `loadFlatConfigFile` should be very easy to restore with our current knowledge, so I'd say it's not necessary to keep comments in production code.